documentation/docs/backend/email.md

```diff
@@ -1,5 +1,15 @@
 # Email
 
-MailHog is being used to test the sending of emails. It is an email-testing tool with a fake SMTP server underneath. To view the emails which have been sent, open `http://localhost:8025/` in your browser, which will open the MailHog user interface.
+### Development
+
+MailHog is being used to test the sending of emails for free in a development environment. It is an email-testing tool with a fake SMTP server underneath. To view the emails which have been sent, open `http://localhost:8025/` in your browser, which will open the MailHog user interface.
 
 If you haven't got it installed already, Mailhog should automatically install when you run `docker compose up`.
+
+### Production
+
+Sendgrind is used to send and recieve emails in the production version of the application. Community Spirit Foundation email account has been added as the sender. The Sendgrid account is made under Coders For Causes credentials and will allow for 100 free emails per day, anything over will cost $20 per month.
```

Comment:
- Isn't it "Sendgrid" not "Sendgrind" xD 
- Add "The" before  "Community Spirit Foundation email account"

Reply:
Just noticed another "recieve" which should be "receive"